.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish to Github Packages
```

Comment:
This publish-package.yml file duplicates what's done in [main.yml](https://github.com/polarislabs/pg-to-dbml/blob/develop/.github/workflows/main.yml). 
Can you maybe delete this one, and rename the existing main.yml to publish-package.yml?